Gulpfile.js

```diff
@@ -10,14 +10,13 @@ const filter = require("gulp-filter");
 const gulp = require("gulp");
 const path = require("path");
 const rollup = require("rollup");
-const rollupBabel = require("rollup-plugin-babel");
+const rollupBabel = require("@rollup/plugin-babel").default;
 const rollupBabelSource = require("./scripts/rollup-plugin-babel-source");
-const rollupCommonJs = require("rollup-plugin-commonjs");
+const rollupCommonJs = require("@rollup/plugin-commonjs");
 const rollupJson = require("@rollup/plugin-json");
-const rollupNodeBuiltins = require("rollup-plugin-node-builtins");
-const rollupNodeGlobals = require("rollup-plugin-node-globals");
-const rollupNodeResolve = require("rollup-plugin-node-resolve");
-const rollupReplace = require("rollup-plugin-replace");
+const rollupNodePolyfills = require("rollup-plugin-node-polyfills");
```

Comment:
`rollup-plugin-node-polyfills` replaces both `rollup-plugin-node-globals` and `rollup-plugin-node-builtins`. This has been done on v8 branch and now can be backported since we are not building on node.js 6.